static/js/tools/map/bq_query_utils.ts

```diff
@@ -180,8 +183,10 @@ function getBaseWhereClause(
 }
 
 export function getPcQuery(
```

Comment:
looks like the BQ modal is in many tools not only map. Should these live outside the map folder?

Reply:
The BQ modal lives in the shared folder. These are the query utils specific to getting the query for the map tool!